processor/sumologicprocessor/README.md

```diff
@@ -0,0 +1,324 @@
+
+[sumologic_webpage]: https://www.sumologic.com
+[sumologic_apps]: https://www.sumologic.com/applications/
+
+## Configuration
+
+```yaml
+processors:
+  sumologic:
+    # Defines whether the `cloud.namespace` resource attribute should be added.
+    # default = true
+    add_cloud_namespace: {true,false}
+
+    # Defines whether attributes should be translated
+    # from OpenTelemetry to Sumo Logic conventions;
+    # see "Attribute translation" documentation chapter from this document.
+    # default = true
+    translate_attributes: {true,false}
+
+    # Specifies whether telegraf metric names should be translated to match
+    # Sumo Logic conventions expected in Sumo Logic host related apps (for example
+    # `procstat_num_threads` => `Proc_Threads` or `cpu_usage_irq` => `CPU_Irq`).
+    # See `translate_telegraf_metrics_processor.go` for full list of translations.
+    # default = true
+    translate_telegraf_attributes: {true, false}
+
+    # Specifies whether docker stats metric names should be translated to match
+    # Sumo Logic conventions expected in Sumo Logic host related apps (for example
+    # `container.cpu.usage.kernelmode` => `cpu_usage.usage_in_kernelmode` or `container.memory.usage.max` => `max_usage`).
+    # It also translates some resource attribute names (for example `container.id` => `container.FullID`).
+    # See `translate_docker_metrics_processor.go` for full list of translations.
+    # default = false
+    translate_docker_metrics: {true, false}
+
+    # Specifies if attributes should be nested, basing on their keys.
+    # See "Nesting attributes" documentation chapter from this document.
+    nest_attributes:
+      # Defines whether attributes should be nested.
+      # default = false
+      enabled: {true, false}
+
+      # Defines the string used to separate key names in attributes that are to be nested.
+      # default = "."
+      separator: <separator>
+
+      # Defines a list of allowed prefixes to be nested.
+      # For example, if "kubernetes." is in this list, then all "kubernetes.*" attributes will be nested.
+      # default = []
+      include: [<prefix>]
+
+      # Defines a list of prefixes not allowed to be nested.
+      # For example, if "k8s." is in this list, then all "k8s.*" attributes will not be nested.
+      # default = []
+      exclude: [<prefix>]
+
+      # If enabled, then maps that would have only one value will be squashed.
+      # For example,{"k8s": {"pods": {"a": "A", "b": "B"}}}
+      # will be squashed to {"k8s.pods": {"a": "A", "b": "B"}}
+      # default = false
+      squash_single_values: {true, false}
+
+    # Specifies if attributes matching given pattern should be mapped to a common key.
+    # See "Aggregating attributes" documentation chapter from this document.
+    # default = []
+    aggregate_attributes:
+      - attribute: <attribute>
+        prefixes: [<prefix>]
+      - ...
+
```

Comment:
For the record: these two features were created as a temporary solution for these issues: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14977 https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14946

Reply:
1. Mark configuration as deprecated and add warning in case of using it and also instruction how to use new configuration
2. Remove support for the functionality, but leaving logs in order to avoid confusion with default missing configuration error
3. Remove logs and configuration option completely